controllers/openshift_controller.go

```diff
@@ -1731,6 +1731,33 @@ func (r *KataConfigOpenShiftReconciler) enablePeerPods() error {
 		return err
 	}
 
+	pullSecret := &corev1.Secret{}
```

Comment:
Consider encapsulate the everything in a function (to keep it clean with the latest mc functions that has been added)